metaseq/models/transformer_lm.py

```diff
@@ -132,6 +132,12 @@ class TransformerLanguageModelConfig(MetaseqDataclass):
         default=False,
         metadata={"help": "Exact same init as Megatron"},
     )
+    full_megatron_init_scalar: float = field(
```

Comment:
Nit: An arg name that includes second_layer makes might be more understandable: `full_megatron_init_scalar_second_layer`

Reply:
It includes MHA out proj so.. it's sadly kind of misnamed either way :(